T/tblis/build_tarballs.jl

```diff
@@ -8,23 +8,90 @@ version = v"1.2.0"
 # Collection of sources required to complete build
 sources = [
     GitSource("https://github.com/devinamatthews/tblis.git", "3e4c4b82943726c443b6f408c9c9791dcad7a847")
+    DirectorySource("./bundled")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir
 cd tblis/
-./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+
+for i in ./Makefile.* ./configure*; do
+
+    # Building in container forbids -march options
+    sed -i "s/-march[^ ]*//g" $i
+
+done
+
+case ${target} in
+    # Unlike stated in Wiki, 
+    # TBLIS automatically detects threading model.
+    *"x86_64"*"linux"*"gnu"*) 
+        export BLI_CONFIG=x86
+        export BLI_THREAD=openmp
+        ;;
+    *"x86_64"*"linux"*"musl"*)
+        export BLI_CONFIG=x86
+        export BLI_THREAD=pthreads
+        export CC=clang
+        export CXX=clang++
+        ;;
+    *"x86_64"*"w64"*)
+        # Windows lacks support for some instructions.
+        # Building only for AMD processors.
+        export BLI_CONFIG=amd
+        export BLI_THREAD=openmp
+        # Wrapper for posix_memalign calls.
+        patch src/memory/aligned_allocator.hpp < ${WORKSPACE}/srcdir/patches/aligned_allocator.hpp.mingw.patch
+        # Additional linking parameter needed for MinGW Autoconf.
+        # Update Autoconf parameters and refresh.
+        cd src/external/tci
+        echo "AM_LDFLAGS=-no-undefined" >> Makefile.am
+        echo "lib_libtci_la_LDFLAGS=-no-undefined" >> Makefile.am
+        update_configure_scripts --reconf
+        cd ../../..
+        sed -i "s/.*AM_LDFLAGS.*/AM_LDFLAGS=-lpthread -no-undefined/" Makefile.am
+        echo "lib_libtblis_la_LDFLAGS=-no-undefined" >> Makefile.am
+        update_configure_scripts --reconf
+        ;;
+    *"x86_64"*"apple"*) 
+        export BLI_CONFIG=x86
+        export BLI_THREAD=openmp
+        export CC=gcc
+        export CXX=g++
+        ;;
+    *"x86_64"*"freebsd"*) 
+        export BLI_CONFIG=x86
+        export BLI_THREAD=openmp
+        export CC=gcc
+        export CXX=g++
+        ;;
+    *)
+        ;; 
+
+esac
+
+CFG_OPTION_POSIX="--prefix=${prefix} --build=${MACHTYPE} --host=${target}"
+# ./configure will warn about --enable-thread-model but it is actually effective.
+CFG_OPTION_TBLIS="--enable-config=${BLI_CONFIG} --enable-thread-model=${BLI_THREAD}"
+
+./configure ${CFG_OPTION_TBLIS} ${CFG_OPTION_POSIX}
 make -j${nproc}
-cp lib/.libs/libtblis.so.0.0.0 ${libdir}/libtblis.so.0
-cp src/external/tci/lib/.libs/libtci.so.0.0.0 ${libdir}/libtci.so.0
+make install
+
+# Copy license file
+mkdir -p ${prefix}/share/licenses/tblis
+cp LICENSE ${prefix}/share/licenses/tblis
```

Comment:
We have [a tool to do this automatically](https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#Installing-the-license-file-1)
```suggestion
install_license LICENSE
```